src/main/java/post_reply_user/Post.java

```diff
@@ -0,0 +1,104 @@
+package post_reply_user;
+
+import java.text.SimpleDateFormat;
+import java.util.ArrayList;
+import java.util.Date;
+
+public class Post {
+    common_User auther;
+    String content;
+    int likes;
+    String time;
+    ArrayList<common_User> replyedBy;
+    ArrayList<Reply> totalReply;
+    ArrayList<common_User> likedBy;
+
+    public void setAuther(common_User auther) {
+        this.auther = auther;
+    }
+
+    public void setContent(String content) {
+        this.content = content;
+    }
+
+    public void setTime() {
+        this.time = String.valueOf(new Date());
+        SimpleDateFormat dateFormat = new SimpleDateFormat("yyyy-MM-dd hh:mm:ss");
```

Comment:
It would be better to get the time data from the Database, and you could write a method that converts the Database time to a simple time format.

Reply:
I am not so sure cause Database will store the time anyway.